resources/quality/xyzprinting/xyzprinting_da_vinci_jr_pro_xeplus_global_0.40_coarse.inst.cfg

```diff
@@ -0,0 +1,27 @@
+[general]
+version = 4
+name = Coarse Quality
+definition = xyzprinting_da_vinci_jr_pro_xeplus
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = coarse
+weight = -3
+global_quality = True
+
+[values]
+adhesion_type = brim
+brim_width = 10.0
+cool_fan_speed = 100
```

Comment:
It's really strange to set this in the global quality. In most cases the cool_fan_speed is very much material dependant!

Also, if it's always 100, why not just set that in the definition. It keeps you from repeating yourself over and over again (DRY principle: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
OK,We will move them to XYZprinting_base.
Thank you :)